lib/src/main/java/io/cloudquery/server/PackageCommand.java

```diff
@@ -0,0 +1,299 @@
+      }
+    }
+  }
+
+  private void copyDocs() throws IllegalArgumentException, IOException {
+    File docs = new File(docsDir);
+    if (!docs.exists()) {
+      logger.error("Docs directory path{} does not exist", docsDir);
+      throw new IllegalArgumentException("Docs directory does not exist");
+    }
+    if (!docs.isDirectory()) {
+      logger.error("Docs path {} is not a directory", docsDir);
+      throw new IllegalArgumentException("Docs path is not a directory");
+    }
+
+    String outputPath = Paths.get(distDir, "docs").toString();
+    logger.info("Copying docs from {} to {}", docsDir, outputPath);
+    File output = new File(outputPath);
+    FileUtils.copyDirectory(docs, output);
+  }
+
+  private void writeTablesJson() throws Exception {
+    if (plugin.getKind() != PluginKind.Source) {
+      return;
+    }
+
+    String outputPath = Paths.get(distDir, "tables.json").toString();
+    logger.info("Writing tables.json to {}", outputPath);
+    plugin.init("", NewClientOptions.builder().noConnection(true).build());
+    List<Table> tables = plugin.tables(Arrays.asList("*"), Arrays.asList(), false);
+    List<Table> flattenTables = Table.flattenTables(tables);
+    TablesJson tablesJson = new TablesJson(flattenTables);
+    String json = tablesJson.toJson();
+    FileUtils.writeStringToFile(new File(outputPath), json, "UTF-8");
+  }
+
+  @SuppressWarnings("null")
+  private List<SupportedTargetJson> buildDocker()
+      throws IllegalArgumentException, IOException, InterruptedException {
+    String dockerFilePath = Paths.get(pluginDirectory, plugin.getDockerfile()).toString();
+    File dockerFile = new File(dockerFilePath);
+    if (!dockerFile.exists()) {
+      logger.error("Dockerfile {} does not exist", dockerFilePath);
+      throw new IllegalArgumentException("Dockerfile does not exist");
+    }
+    if (!dockerFile.isFile()) {
+      logger.error("Dockerfile {} is not a file", dockerFilePath);
+      throw new IllegalArgumentException("Dockerfile is not a file");
+    }
+
+    List<SupportedTargetJson> supportedTargets = new ArrayList<>();
+    for (BuildTarget target : plugin.getBuildTargets()) {
+      String imageRepository =
+          String.format(
+              "docker.cloudquery.io/%s/%s-%s",
+              plugin.getTeam(), plugin.getKind(), plugin.getName());
+      String os = target.getOs().toString();
+      String arch = target.getArch().toString();
+      String imageTag = String.format("%s:%s-%s-%s", imageRepository, pluginVersion, os, arch);
+      String imageTar =
+          String.format("plugin-%s-%s-%s-%s.tar", plugin.getName(), pluginVersion, os, arch);
+      String imagePath = Paths.get(distDir, imageTar).toString();
+      logger.info("Building docker image {}", imageTag);
+      // GITHUB_ACTOR and GITHUB_TOKEN are required for the Dockerfile to pull the CloudQuery Java
+      // libs from GitHub Packages
+      String githubActor = System.getenv("GITHUB_ACTOR");
```

Comment:
There's a tricky part with Java since we host the SDK on GitHub packages which require a token to download libs, so we read it from env variables